.github/workflows/pr-verification.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: PR verification
+on:
+  pull_request:
+
+env:
+  GOPATH: /home/runner/go
+  GOCACHE: /home/runner/go/cache
+  GOMODCACHE: /home/runner/go/mod
+  GOPROXY: https://proxy.golang.org # remove direct
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-20.04
```

Comment:
I think we should also experiment with

```yaml
concurrency: 
  cancel-in-progress: true
```

especially if we are going to use `edited` activity type

Reply:
Added